### PR TITLE
update readme and contribute docs

### PR DESCRIPTION
**Pull request by nabuskey**
- Move some of development stuff to contribution doc
- Update readme to reflect latest changes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -54,17 +54,127 @@ UIs for Backstage, Gitea, and ArgoCD are accessible on the machine:
...
+- ArgoCD initial admin password.
+- Gitea admin user credentials.
+- Any secrets labeled with `cnoe.io/cli-secret=true`.
+
+You can think of the command as executing the following kubectl commands:
+
+```bash
+kubectl -n argocd get secret argocd-initial-admin-secret
+kubectl get secrets -n gitea gitea-admin-secret
+kubectl get secrets -A -l cnoe.io/cli-secret=true
```
Comment (nimakaviani) [inline CONTRIBUTING.md:74]: doesn't this belong to the main README? Independent of whether you want to contribute or not, this is still very handy for users.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -54,17 +54,127 @@ UIs for Backstage, Gitea, and ArgoCD are accessible on the machine:
...
+This repository requires a [Developer Certificate of Origin (DCO)](https://developercertificate.org/) signature. 
+When preparing to send in a pull request, please make sure your commit is signed. You can achieve this by doing a `git commit --sign` or `git commit -s` when making the commit.
+
+## Project Information
+
+### Default manifests installed by idpbuilder
+
+The default manifests for the core packages are available [here](pkg/controllers/localbuild/resources).
+These are generated by scripts. If you want to make changes to them, see below.
+
```
Comment (nimakaviani) [inline CONTRIBUTING.md:93]: good to call out the section in the README where the embedded versions of these packages are listed.

**Review by nimakaviani — COMMENTED** [review]
two minor comments. otherwise looks great. thanks for this!

**Comment by nabuskey** [comment]
Updated docs again. Please have another pass.

**Pull request merged**